### PR TITLE
in the resize operation halves were not resized

### PR DESCRIPTION
**Pull request by Vilax**

### CODE REVIEW

File: relion/protocols/protocol_crop_resize.py [hunk 1]
```diff
@@ -62,7 +64,7 @@ def _defineParams(self, form):
         form.addParam('rescaleSamplingRate', params.FloatParam,
                       default=1.0,
                       condition='doRescale',
-                      label='New sampling rate (Å/px)')
+                      label='New sampling rate (A/px)')
```
Comment (JorMaister) [inline relion/protocols/protocol_crop_resize.py:67]: Just for curiosity, what's the motivation of change? Is the Angstrom symbol not being displayed properly somewhere?

File: relion/protocols/protocol_crop_resize.py [hunk 2]
```diff
@@ -93,17 +95,34 @@ def convertInputStep(self):
...
         if self.doResize:
             argDict[' --new_box'] = self.resizeSize.get()
 
-        for i, fn in enumerate(self.convertedVols):
+        if isinstance(inputData, Volume):
+            self.runResizeCmd(self.convertedVols, argDict)
+            if inputData.hasHalfMaps():
+                halves = inputData.getHalfMaps().split(',')
+                self.convertedVols = []
+                print(halves)
```
Comment (JorMaister) [inline relion/protocols/protocol_crop_resize.py:112]: Is this forgotten debugging code? If not, I suggest to use the logger functionality or the Protocol info method.

**Review by JorMaister — CHANGES_REQUESTED** [review]
Please have a look at the comments I wrote (I did not notice I was not in the review mode when I added them)

File: relion/protocols/protocol_crop_resize.py (hunk 1 above)
Comment (Vilax) [reply relion/protocols/protocol_crop_resize.py:67]: No motivation, a ? symbol apears in my computer, not the A symbol. So I changed it. But it was ok. I will change it as it was

File: relion/protocols/protocol_crop_resize.py (hunk 2 above)
Comment (Vilax) [reply relion/protocols/protocol_crop_resize.py:112]: My bad

**Review by JorMaister — APPROVED** [review]

**Comment by azazellochg** [comment]
🤦‍♂️why to master

**Comment by azazellochg** [comment]
You didn't even bump the version, so these changes are not released anywhere

**Comment by JorMaister** [comment]
My bad, I didn't notice it was to master. I'll fix it.

**Comment by JorMaister** [comment]
I have the revision hash located but don't have permission to directly push to master to undo the merge. I've asked @pconesa to do it. Meanwhile, I restored the branch so the PR can be done properly to devel. My apologies, @azazellochg

**Comment by azazellochg** [comment]
No problem, just more work for yourself 😉

**Comment by JorMaister** [comment]
Hi, @azazellochg, 
I've been talking to @pconesa and we've decided to send the same changes to devel as it's a very small changeset to prevent both devel and master from diverging. Are you ok with that?

**Comment by azazellochg** [comment]
Hi,

Of course! Go ahead

**Pull request merged**